docs/quickstart.md

```diff
@@ -148,9 +148,9 @@ Status:
 Events:
   Type    Reason           Age   From       Message
   ----    ------           ----  ----       -------
-  Normal  CreateSucceeded  11s   kubestone  Created /api/v1/namespaces/kubestone/configmaps/fio-sample
-  Normal  CreateSucceeded  11s   kubestone  Created /api/v1/namespaces/kubestone/persistentvolumeclaims/fio-sample
-  Normal  CreateSucceeded  11s   kubestone  Created /apis/batch/v1/namespaces/kubestone/jobs/fio-sample
+  Normal  Created  11s   kubestone  Created /api/v1/namespaces/kubestone/configmaps/fio-sample
```

Comment:
this reminds me that kubernetes 1.16 deprecates selfLink, which is used here to report the object name. 
Later (in a seperate ticket), we need to fix this.